src/screens/Home/HomeScreen/HomeScreen.tsx

```diff
@@ -12,48 +12,48 @@ import {
   ImageBackground,
 } from 'react-native'
 import styles from './styles'
-
-// questions
-const DATA = [
-  {
-    id: '1',
-    title: 'Who/what influenced or inspired you to do what you do?',
-  },
-  {
-    id: '2',
-    title: 'What is a typical day for you?',
-  },
-  {
-    id: '3',
-    title: 'What do you love most about your job?',
-  },
-  {
-    id: '4',
-    title: 'hi',
-  },
-  {
-    id: '5',
-    title: 'hi',
-  },
-]
+import { BASE_PATH } from 'react-native-dotenv'
+ 
+interface IQuestion {
+  id: string;
+  text: string;
+}
 
 /* AKA: Q&A screen */
-export default function HomeScreen() {
-  const [selected, setSelected] = useState()
+export default function HomeScreen() { 
+  const [selected, setSelected] = useState<string | null>(null)
+  const [questions, setQuestions] = useState<Array<IQuestion>>([])
+
+  useEffect(() => {
+    fetch(`${BASE_PATH}/api/questions`)
+      .then(res => res.json())
+      .then(data => {
+        setQuestions( data )
+        console.log(data)
+      })
+      .catch(error => {
+        console.log('Error' + error)
+      })
+  }, [])
 
   return (
     <SafeAreaView style={styles.container}>
       <ImageBackground source={background} style={{width: '100%', height: '100%'}}>
         <View style={styles.banner}>
           <Text style={styles.bannertext}>QUESTIONS</Text>
-          <Image source={logo} style={styles.bannerlogo} />
+          <View style = {styles.counter}>
+              <View style = {styles.numberCounter}>
+                <Text style={styles.number}>0</Text>
+              </View>
+              <Text style={styles.answered}>answered</Text>
+            </View>
         </View>
```

Comment:
This header section is actually in the true header of the screen in the figma. This is set with react navigation options. Below your function you can write something like this. (you may have to fiddle w/ the styling)

```js
HomeScreen.navigationOptions = {
  header: <Text style={styles.bannertext}>QUESTIONS</Text>,
  headerRight: 
    <View style = {styles.counter}>
      <View style = {styles.numberCounter}>
        <Text style={styles.number}>0</Text>
      </View>
      <Text style={styles.answered}>answered</Text>
    </View>
}
```

Reply:
bump